### ISSUE TITLE
Error (code 6) on "eune"

### ISSUE THREAD

**Issue by WildEgo** [supergrecko/RiotQuest#51]
**Describe the bug**
When trying to set the region to eune I get thrown an error code 6

**To Reproduce**
1. LeagueApi::summoner('eune')->name($result->account_ign)
2. "ERROR (code 6): Specified region could not be resolved."

**Expected behavior**
I'm supposed to receive summoner information .

**Desktop (please complete the following information):**
 - OS: Windows 10
 - PHP Version: 7.2.10
 - RiotQuest Version: 1.3.4

**Comment by supergrecko** [comment]
Hi, eune does currently not resolve into a valid region (https://github.com/supergrecko/RiotQuest/blob/master/src/RiotQuest/Components/Engine/Utils.php#L34)

I'll update it so it'll resolve.

**Comment by WildEgo** [comment]
Thanks could you do the same check up for the other regions? I'd really appreciate it that way in case it happens again I don't post here, also if you don't mind how can I handle exceptions? Since they're just throwing a complete error page instead of something I can work with

**Comment by supergrecko** [comment]
You can catch the LeagueException type. I'm going to create a more intuitive exception handling system for the next major release which will be a while after PHP 7.4's release.

**Comment by WildEgo** [comment]
Sorry for being annoying but could you give me a snippet just to check if what I wrote is wrong or not (I'm using Laravel in case it helps)

**Comment by supergrecko** [comment]
Hello, this code snippet works perfectly fine for me on 1.3.4

```php
dump(Client::summoner('eune', 1)->name('abc'));
```

And to answer your second question:

```php
use RiotQuest\Contracts\LeagueException;

try {
    dump(Client::summoner('eune', 1)->name('abc'));
} catch (LeagueException $ex) {
    // deal with $ex
}

Is there anything on your end which could be breaking it or are you passing the literal `'eune'` string?

**Comment by WildEgo** [comment]
Oh I think I found the problem the method `->getRanked()` is the one throwing the exception it looks like

**Comment by supergrecko** [comment]
This has been resolved, you can grab it via `1.3.5`
